### DISCUSSION TITLE
Datasource artifactory_file with path_is_aliased

### DISCUSSION THREAD

**Discussion by qtruong77** [jfrog/terraform-provider-artifactory#572] [Q&A] [answered]
Hi,
I have an issue when using provider with Artifactory behind Nginx, where it trims the "artifactory" top level in URL path. I use the configuration as below

```
provider "artifactory" {
  url = "https://artifactory.company.com"
  check_license = false
  api_key = "ABCXYZ"
}
data "artifactory_file" "source_file" {
  repository  = "my-repo"
  path        = "path/to/file"
  path_is_aliased = true
  force_overwrite = true
  output_path = "local-file.zip"
}
```
Then I got 404 error
```
Error: 
│ 404 GET https://artifactory.company.com/artifactory/my-repo/path/to/file
```

Where I can use `curl -H "X-JFrog-Art-Api:ABCXYZ" -O "https://artifactory.company.com/my-repo/paht/to/file"` to download file. Not sure what I missed here.

**Comment by alexhung** [comment, marked as answer]
The 'artifactory' alias to sub-domain is not recommended since v7(?). Even though you can still kinda use it, it really will break many APIs. Before you mention it, I know the JFrog CLI accommodates this alias and there has been calls to support this in the provider. See https://github.com/jfrog/terraform-provider-artifactory/issues/515

Please comment in that issue so we get a sense of how many users are requesting this feature.
